test/common/chromium-cdp-driver.js

```diff
@@ -279,60 +274,63 @@ function setupSSLCertHandling(client) {
  *    fail <JSON formatted error>
  * EOF shuts down the client.
  */
-process.stdin.setEncoding('utf8');
-
-if (process.env.TEST_CDP_DEBUG)
-    enable_debug = true;
-
-const options = { };
-if (process.argv.length >= 3) {
-    options.port = parseInt(process.argv[2]);
-    if (!options.port) {
-        process.stderr.write("Usage: chromium-cdp-driver.js [port]\n");
-        process.exit(1);
+async function main() {
+    process.stdin.setEncoding('utf8');
+
+    if (process.env.TEST_CDP_DEBUG)
+        enable_debug = true;
+
+    const options = { };
+    if (process.argv.length >= 3) {
+        options.port = parseInt(process.argv[2]);
+        if (!options.port) {
+            process.stderr.write("Usage: chromium-cdp-driver.js [port]\n");
+            process.exit(1);
+        }
     }
+
+    const target = await CDP.New(options);
+    target.port = options.port;
+    const client = await CDP({ target });
+    setupLogging(client);
+    setupFrameTracking(client);
+    setupSSLCertHandling(client);
+    setupLocalFunctions(client);
+
+    let input_buf = '';
+    process.stdin
```

Comment:
Couldn't we do something more like `await process.stdin.read()` in a loop or so?

Reply:
Apparently that looks like `for await (const chunk of process.stdin) {`